README.md

```diff
@@ -1 +1,133 @@
-# decode-morse-code
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Morse Decoder <a name="about-project"></a>
+
+**Morse Decoder** A Morse Decoder is used to decode the Morse code to an actual readable word or message.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby.rb</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/ruby">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Relational Database**
+- **Uses Synthetic Primary Key
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby installer
+- Code Editor
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+cd my-folder
+git clone git@github.com:firepicaso/decode-morse-code.git
+```
+
+### Install
+
+Install this project with:
+No installation needed
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **FirePicaso**
+
+- GitHub: [@githubhandle](https://github.com/firepicaso)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mustakim-masum/)
+
+👤 **Albert Antwi**
+
+- GitHub: [@githubhandle](https://github.com/albertkantwi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/albertkantwi/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Morse Decoder**
+- [ ] **more robust ruby functions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/firepicaso/decode-morse-code/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
```

Comment:
- [ Required ] As suggested by the previous reviewer, I think it would be nice if you wrote a descriptive message in this part of the readme file. The current text is not descriptive, which makes your readme file look unprofessional. Please write a descriptive message of support.